src/mappings/pricing.ts

```diff
@@ -19,25 +24,6 @@ export function getEthPriceInUSD(): BigDecimal {
   }
 }
 
-// token where amounts should contribute to tracked volume and liquidity
-let WHITELIST: string[] = [
-// TODO: update address
-  '0x82af49447d8a07e3bd95bd0d56f35241523fbab1', // WETH
-  '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8', // USDC.e
-  '0xaf88d065e77c8cc2239327c5edb3a432268e5831', // USDC
-  '0x912ce59144191c1204e64559fe8253a0e49e6548', // ARB
-  '0xd74f5255d557944cf7dd0e45ff521520002d5748', // USDs
-  '0x1622bf67e6e5747b81866fe0b85178a93c7f86e3', // UMAMI
-  '0x6c2c06790b3e3e3c38e12ee22f8183b37a13ee55', // DPX
-  '0x5979d7b546e38e414f7e9822514be443a4800529', // wstETH
-  '0x6cda1d3d092811b2d48f7476adb59a6239ca9b95', // stafi-rETH
-  '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9', // USDT
-  '0xda10009cbd5d07dd0cecc66161fc93d7c9000da1', // DAI
-  '0xeb1a8f8ea373536600082ba9ae2db97327513f7d', // apOHM
-]
-
-let STABLE = '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8'
-
 // minimum liquidity required to count towards tracked volume for pairs with small # of Lps
 let MINIMUM_USD_THRESHOLD_NEW_PAIRS = BigDecimal.fromString('500')
```

Comment:
Thresholds should be dynamically set too

Reply:
`MINIMUM_USD_THRESHOLD_NEW_PAIRS`, `MINIMUM_LIQUIDITY_THRESHOLD_ETH` and `MINIMUM_LIQUIDITY_ETH` won't be the same for each chain